File: linux/ardium_source/presentation/01_Basics/basics.go
// Topic: Basic Control Flow
// Count 0-9 and identify Even/Odd

package main

import "fmt"

func main() {
    i := 0
    for i < 10 {
        if i % 2 == 0 {
            fmt.Printf("Count: %d (Even)\n", i)
        } else {
            fmt.Printf("Count: %d (Odd)\n", i)
        }
        i++
    }
}

File: linux/ardium_source/presentation/01_Basics/basics.m
// Topic: Basic Control Flow
// Count 0-9 and identify Even/Odd

#import <Foundation/Foundation.h>

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        int i = 0;
        while (i < 10) {
            if (i % 2 == 0) {
                NSLog(@"Count: %d (Even)", i);
            } else {
                NSLog(@"Count: %d (Odd)", i);
            }
            i++;
        }
    }
    return 0;
}
